src/predict/jobs/forecast/Dockerfile

```diff
@@ -14,13 +14,16 @@ RUN pip install -r requirements.txt
 FROM base as production
 ENV ENV=production
 CMD ["python", "daily_forecasts.py"]
+CMD ["python", "hourly_forecasts.py"]
 
 # Staging
 FROM base as staging
 ENV ENV=staging
 CMD ["python", "daily_forecasts.py"]
+CMD ["python", "hourly_forecasts.py"]
```

Comment:
Hi @NoahNsimbe and @123MwanjeMike. I need your help here. These 2 scripts are separate jobs and ought to run at different times. Is this okay? How does one discern between them which one to run at a given period

Reply:
@123MwanjeMike

Reply:
Hi @Mnoble-19 
Why are you having two different entry points(`CMD`) in the same stage dockerfile.
Can we use different stages for this.
https://docs.docker.com/build/building/multi-stage/